### ISSUE TITLE
Any t-PIT loss calculation included?

### ISSUE THREAD

**Issue by shincling** [yuzhou-git/deep-casa#2]
Seems the loss in current code is based on uPIT.
In your papers, the tPIT seems get obvious better performance. 
Will you add this part? 
另外，Yuzhou不知道可不可以加个微信认识一下？ 我是自动化所，现在在JHU访学。

**Comment by shincling** [comment]
Update: 
Seems I made some misunderstanding,  after reading the code, I fould current version is actually based on tPIT. You treated the fist size of Y1R ( Y1I, Y2R.....) as batch_size*Time. And take each frame in each mixed sample as an independent element to calculate the loss and permutation. So it is the tPIT I guess. Is it right ?
https://github.com/yuzhou-git/deep-casa/blob/90c6e8fc4f821a90a298fe682dd623cc92324a9c/nn/simul_group.py#L335

**Comment by yuzhou-git** [comment]
Yes. The current version uses tPIT loss to reorganize frames, and calculate SNR on an utterance level.
